src/intl/en.json

```diff
@@ -286,7 +286,8 @@
     "active": "Active outcomes",
     "rejected": "Rejected outcomes",
     "passed": "Passed outcomes",
-    "enacted": "Enacted outcomes"
+    "enacted": "Enacted outcomes",
+    "out_of_budget": "OOB outcomes"
```

Comment:
Is this ok with @ginocingolani? Not sure about the acronym here.

Reply:
I believe Guane is going to use the acronym in the pills. But we can change it later :)